pkg/cli/cmds/server.go

```diff
@@ -11,6 +12,9 @@ import (
 
 const (
 	DisableItems = "coredns, servicelb, traefik, local-storage, metrics-server"
+
+	defaultSnapshotRentention      = 5
+	defaultSnapshotIntervalMinutes = 5
```

Comment:
Is 5 minutes really a sensible default? I would think something much higher (6, 12, 24 hours) would make more sense?

Reply:
It does seem a bit low but was part of the initial PR. I figured it was there for a reason. I'm happy to bump that default up to something like you list above.

Reply:
What does rke1 do by default?

Reply:
Would appear to be every 12 hours. Happy to make it that. Seems reasonable.

Reply:
12 hours or 12 minutes?